production/operator/templates/agent-operator.yaml.t

```diff
@@ -0,0 +1,645 @@
+apiVersion: v1
```

Comment:
Ditto with this file: we may be ale to delete it in favor of the operator Helm chart; I'm not sure where this is used at all, actually. I can't find any references to it in the website or in this repo, so it might be safe to remove it.